src/currencyservice/src/server.cpp

```diff
@@ -120,7 +131,7 @@ class CurrencyService final : public hipstershop::CurrencyService::Service
 
     span->AddEvent("Currencies fetched, response sent back");
     span->SetStatus(StatusCode::kOk);
-    // Make sure to end your spans!
+   // Make sure to end your spans!
```

Comment:
I don't see a reason for this indentation change. I think it was better before (aligned with the code).

This comment is valid for all similar changes below.

Reply:
Good catch. I had a space bug I fixed in bulk and this changed too. Resolved